dev/benchmarks/complex_layout/windows/flutter/CMakeLists.txt

```diff
@@ -1,7 +1,3 @@
-# Copyright 2014 The Flutter Authors. All rights reserved.
-# Use of this source code is governed by a BSD-style license that can be
-# found in the LICENSE file.
-
```

Comment:
Any reason for the copyright header removal?

Reply:
yeah, requesting changes on this so we don't accidentally merge this

Reply:
I simply copied the template file to replace all its copies, and it seems like copyright is present on some, and not on others. For now, I just reintegrated copyright where it was, and we can fix this later.